dipy/workflows/tests/test_tracking.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+from nose.tools import assert_false
+from os.path import join
+
+import nibabel as nib
+from nibabel.tmpdirs import TemporaryDirectory
+
+from dipy.data import get_data
+from dipy.io.image import save_nifti
+from dipy.workflows.mask import MaskFlow
+from dipy.workflows.reconst import ReconstCSDFlow
+from dipy.workflows.tracking import DetTrackSHFlow, DetTrackPAMFlow, \
```

Comment:
Looks like maybe `DetTrackSHFlow` is not implemented yet?

Reply:
I removed them from this PR.